src/main/java/org/gridsuite/study/server/networkmodificationtree/NetworkModificationNodeInfoRepositoryProxy.java

```diff
@@ -173,7 +212,8 @@ public void invalidateNodeBuildStatus(AbstractNode node, List<UUID> changedNodes
 
         modificationNode.setNodeBuildStatus(NodeBuildStatus.from(BuildStatus.NOT_BUILT));
         modificationNode.setVariantId(UUID.randomUUID().toString());
-        modificationNode.setReportUuid(UUID.randomUUID());
+        modificationNode.setModificationReports(Map.of(modificationNode.getId(), UUID.randomUUID()));
+        modificationNode.setComputationsReports(Map.of());
```

Comment:
Maybe keep these initializers only in invalidateNodeProper method ? The risk I see is that here you set the map to Map.of() and then you reevaluate it in the function to keep voltage init report, then it only works because it is all in one transaction then the first mutation is not directly commited to the DB, but if we renmove transactionnal annoation at higher level it won't work anymore, so it's a bit unstable